### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 106)

### LINKED ISSUE #3737: [devtools-translate] Russian - What's New in DevTools (Chrome 106)
Post: https://developer.chrome.com/blog/new-in-devtools-106/
Translated by: Sep 23

Translator: @solarrust
Reviewers: @webmaxru, @kateryna-prokopenko

### PR DESCRIPTION
**Pull request by solarrust**
Fixes #3737

The long-awaited translation 😅

### CODE REVIEW

**Comment by jecfish** [comment]
Thanks @solarrust! Over to @webmaxru for review

File: site/ru/_partials/devtools/whats-new.md [hunk 1]
```diff
@@ -16,18 +16,18 @@
...
-* [LCP timings breakdown in the Performance Insights panel](/ru/blog/new-in-devtools-106/#insights)
-* [Auto-generate default name for recordings in the Recorder panel](/ru/blog/new-in-devtools-106/#recorder)
-* [Miscellaneous highlights](/ru/blog/new-in-devtools-106/#misc) -->
+### Chrome 106 {: #chrome106 }
+
+* [Группировка по авторским и развёрнутым файлам во вкладке Источники (Sources)](/ru/blog/new-in-devtools-106/#authored)
+* [Связанные трассировки стека для асинхронных операций](/ru/blog/new-in-devtools-106/#async)
+* [Автоматическое игнорирование известных сторонних скриптов](/ru/blog/new-in-devtools-106/#auto-ignore)
+* [Улучшен стек вызовов во время отладки](/ru/blog/new-in-devtools-106/#call-stack)
+* [Скрытые источников из списка игнорируемых во вкладке Источники (Sources)](/ru/blog/new-in-devtools-106/#ignore-nav)
```
Comment (webmaxru) [inline site/ru/_partials/devtools/whats-new.md:25]:
```suggestion
* [Скрытие источников из списка игнорируемых во вкладке Источники (Sources)](/ru/blog/new-in-devtools-106/#ignore-nav)
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 2]
```diff
@@ -22,25 +21,32 @@ draft: true
 {% YouTube id='5gBqTXctxO8' %}
 
 <!-- ## Group files by Authored / Deployed in the Sources panel {: #authored } -->
+## Группировка по авторским и развёрнутым файлам во вкладке Источники (Sources) {: #authored }
 
 <!-- The **Group files by Authored / Deployed** is now shown under the 3-dot menu. Previously, it showed directly on the navigation pane. -->
+Опция **Сгруппировать по авторским и развёрнутым** (Group files by Authored / Deployed) теперь показывается по клику на иконку меню &nbsp; {% Img src="image/admin/4sdCQbpBaG4MpoHB1J08.png", alt="More", width="4", height="20" %}. Раньше она отображалась прямо на панели навигации.
 
 <!-- Open this [demo](https://ng-devtools.netlify.app/). Enable the **Group files by Authored / Deployed** setting to view your original source code (Authored) first and navigate to them quicker. -->
+Откройте это [демо](https://ng-devtools.netlify.app/). Включите опцию **Сгруппировать по авторским и развёрнутым** чтобы видеть ваш оригинальный (Авторский) исходный код в начале и проще по нему навигироваться.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:30]:
```suggestion
Откройте это [демо](https://ng-devtools.netlify.app/). Включите опцию **Сгруппировать по авторским и развёрнутым**, чтобы видеть ваш оригинальный (Авторский) исходный код в начале и проще по нему перемещаться.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 3]
```diff
@@ -52,124 +58,156 @@ async increment() {
 ```
 
 <!-- Previously, the stack trace only showed the timeout operation. It did not show the “root cause” of the operation.  -->
+Раньше трассировка стека показывала только операцию таймаута. Она не показывала корень операции.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:61]:
```suggestion
Раньше трассировка стека показывала только операцию таймаута. Она не показывала первопричину операции.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 4]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
 <!-- With the latest changes, DevTools now shows the operation originates from the `onClick` event in the button component, then the `increment` function, followed by the timeout operation. -->
+Благодаря последним изменениям DevTools теперь показывают всю операцию начиная с события `onClick` на кнопке компонента, затем функцию `increment`, за которой следует функция таймаута.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:64]:
```suggestion
Благодаря последним изменениям, DevTools теперь показывают всю операцию начиная с события `onClick` на кнопке компонента, затем функцию `increment`, за которой следует функция таймаута.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 5]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/2jAETpw8QWzsg1Wqk0Ya.png", alt="Linked stack traces for asynchronous operations", width="800", height="442" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/2jAETpw8QWzsg1Wqk0Ya.png", alt="Связанные трассировки стека для асинхронных операций", width="800", height="442" %}
 
 <!-- Behind the scenes, DevTools introduced a new “Async Stack Tagging” feature. You can tell the whole story of the operation by linking both parts of the async code together with the new `console.createTask()` method. See [Modern debugging in DevTools](/blog/devtools-modern-web-debugging/#linked-stack-traces) to learn more.  -->
+Под капотом этого улучшения скрывается новая возможность “Async Stack Tagging”, представленная в DevTools. Вы можете рассказать полную историю операции при помощи перелинковки обеих частей асинхронного кода с помощью нового метода `console.createTask()`. Прочитайте [Modern debugging in DevTools](/blog/devtools-modern-web-debugging/#linked-stack-traces) чтобы узнать больше.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:69]:
```suggestion
Под капотом этого улучшения скрывается новая возможность “Async Stack Tagging”, представленная в DevTools. Вы можете рассказать полную историю операции при помощи связывания обеих частей асинхронного кода с помощью нового метода `console.createTask()`. Прочитайте [Modern debugging in DevTools](/blog/devtools-modern-web-debugging/#linked-stack-traces), чтобы узнать больше.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 6]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
 
 <!-- ### Automatically ignore known third-party scripts {: #auto-ignore } -->
+### Автоматическое игнорирование известных сторонних скриптов {: #auto-ignore }
 
 <!-- Identify issues in your code quicker during debugging because DevTools now automatically adds known third-party scripts to the ignore list. -->
+Быстрее находите ошибки в своём коде при отладке, теперь DevTools добавляют известные сторонные скрипты в список игнорируемых. 
 
 <!-- Open this [demo](https://ng-devtools.netlify.app/) and click on the increment button. Expand the error message in **Console**. The stack trace shows only your code (e.g. `app.component.ts` `button.component.ts`). Click **Show more frames** to view the full stack trace. -->
+Откройте [демо](https://ng-devtools.netlify.app/) и нажмите на кнопку <kbd>+</kbd>. Разверните ошибку в консоли. Трассировка стека покажет только ваш код (например, `app.component.ts` и `button.component.ts`). Нажмите **Показать ещё фреймы** (Show more frames) чтобы увидеть полную трассировку стека.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:86]:
```suggestion
Откройте [демо](https://ng-devtools.netlify.app/) и нажмите на кнопку <kbd>+</kbd>. Разверните ошибку в консоли. Трассировка стека покажет только ваш код (например, `app.component.ts` и `button.component.ts`). Нажмите **Показать ещё фреймы** (Show more frames), чтобы увидеть полную трассировку стека.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 7]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GQ9B11tKBcFc1BxQYW9z.png", alt="Automatically ignore known third-party scripts in stack trace", width="800", height="425" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/GQ9B11tKBcFc1BxQYW9z.png", alt="Автоматическое игнорирование известных сторонних скриптов при трассировке стека", width="800", height="425" %}
 
 <!-- Previously, the stack trace included third-party scripts like `zone.js` and `core.mjs`. These are not your source code, they are generated by bundlers (e.g. webpack) or frameworks (e.g. Angular). It took a longer time to identify the root cause of an error.  -->
+Раньше трассировка стека включала в себя сторонние скрипты, например, `zone.js` and `core.mjs`. Это не ваш исходный код, они генерируются сборщиками (webpack и другие) или фреймворками (Angular и прочие). Из-за этого определение причины ошибки занимало больше времени.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:91]:
```suggestion
Раньше трассировка стека включала в себя сторонние скрипты, например, `zone.js` and `core.mjs`. Это не ваш исходный код, он генерируются сборщиками (Webpack и другими) или фреймворками (например, Angular). Из-за этого определение причины ошибки занимало больше времени.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 8]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
 <!-- Behind the scenes, DevTools ignores third-party scripts based on the new `x_google_ignoreList` property in sourcemaps. Frameworks and bundlers need to supply this information. See [Case Study: Better Angular Debugging with DevTools](/blog/devtools-better-angular-debugging/#x_google_ignorelist-in-angular).  -->
+По капотом DevTools игнорируют сторонние скрипты при помощи нового свойства `x_google_ignoreList` для карт источников. Фреймворки и сборщики должны предоставлять эту информацию. Читайте [Case Study: Better Angular Debugging with DevTools](/blog/devtools-better-angular-debugging/#x_google_ignorelist-in-angular).
 
 <!-- Optionally, if you prefer to always view full stack traces, you can disable the setting via **Settings** > **Ignore list** > **Automatically add known third-party scripts to ignore list**. -->
+Опционально, если вы предпочитаете видеть полную трассировку стека, то вы можете отключить эту настройку в **Настройки** (Settings) > **Список игнорируемых фреймворков** (Ignore list) > **Автоматически добавлять известные сторонние скрипты в список игнорируемых** (Automatically add known third-party scripts to ignore list).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:99]:
```suggestion
Если же вы предпочитаете видеть полную трассировку стека, то можете отключить эту настройку в **Настройки** (Settings) > **Список игнорируемых фреймворков** (Ignore list) > **Автоматически добавлять известные сторонние скрипты в список игнорируемых** (Automatically add known third-party scripts to ignore list).
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 9]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
 
 <!-- ## Improved call stack during debugging  {: #call-stack } -->
+## Улучшен стек вызовов во время отладки  {: #call-stack }
 
 <!-- With the **Automatically add known third-party scripts to ignore list** setting, the call stack now shows only frames that are relevant to your code. -->
+С настройкой **Автоматически добавлять известные сторонние скрипты в список игнорируемых** (Automatically add known third-party scripts to ignore list) стек вызовов теперь показывает только фреймы, относящиеся к вашему коду.
 
 <!-- Open this [demo](https://ng-devtools.netlify.app/) and set a breakpoint at the `increment()` function in `app.component.ts`. Click the increment button on the page to trigger the breakpoint. The call stack shows only frames from your code (e.g.  `app.component.ts` and `button.component.ts`).  -->
+Откройте это [демо](https://ng-devtools.netlify.app/) и установите точку останова на функции `increment()` в файле `app.component.ts`. Нажмите на кнопку <kbd>+</kbd> на странице чтобы задествовать точку останова. Стек вызовов покажет фреймы только из вашего кода (например,  `app.component.ts` и `button.component.ts`).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:115]:
```suggestion
Откройте это [демо](https://ng-devtools.netlify.app/) и установите точку останова на функции `increment()` в файле `app.component.ts`. Нажмите на кнопку <kbd>+</kbd> на странице, чтобы задействовать точку останова. Стек вызовов покажет фреймы только из вашего кода (например,  `app.component.ts` и `button.component.ts`).
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 10]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
 <!-- To view all frames, enable **Show ignore-listed frames**. Previously, DevTools displayed all frames by default.  -->
+Чтобы увидеть все фреймы нажмите на **Показать фреймы из списка игнорируемых** (Show ignore-listed frames). Раньше DevTools отключал все фреймы по умолчанию.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:118]:
```suggestion
Чтобы увидеть все фреймы, нажмите на **Показать фреймы из списка игнорируемых** (Show ignore-listed frames). Раньше DevTools отключал все фреймы по умолчанию.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 11]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/PdjPrBAV7TXn8FHcRR6R.png", alt="Улучшен стек вызовов во время отладки", width="800", height="601" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/73c559d02676e4329645120e657416e7f15de42b #}
 
-Chromium bug: [1352488](https://crbug.com/1352488)
+Задача в трекере Chromium: [1352488](https://crbug.com/1352488)
 
 
 <!-- ## Hiding ignore-listed sources in the Sources panel {: #ignore-nav } -->
+## Скрытые источников из списка игнорируемых во вкладке Источники (Sources) {: #ignore-nav }
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:128]:
```suggestion
## Скрытие источников из списка игнорируемых во вкладке Источники (Sources) {: #ignore-nav }
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 12]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
 <!-- Enable **hide ignore-listed sources** to hide irrelevant files in the **Navigation** pane. This way, you can focus only on your code. -->
+Включите настроку **Скрыть источники из списка игнорируемых** (Hide ignore-listed sources) чтобы спрятать нерелевантные файлы в панели **Навигация** (Navigation).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:131]:
```suggestion
Включите настроку **Скрыть источники из списка игнорируемых** (Hide ignore-listed sources), чтобы спрятать нерелевантные файлы в панели **Навигация** (Navigation). 
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 13]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
 
 <!-- ## Hiding ignore-listed sources in the Sources panel {: #ignore-nav } -->
+## Скрытые источников из списка игнорируемых во вкладке Источники (Sources) {: #ignore-nav }
 
 <!-- Enable **hide ignore-listed sources** to hide irrelevant files in the **Navigation** pane. This way, you can focus only on your code. -->
+Включите настроку **Скрыть источники из списка игнорируемых** (Hide ignore-listed sources) чтобы спрятать нерелевантные файлы в панели **Навигация** (Navigation). 
 
 <!-- Open this [demo](https://ng-devtools.netlify.app/). In the **Sources** panel. The `node_modules` and `webpack` are the third-party scripts. Click on the 3-dot menu and select **hide ignore-listed sources** to hide them from the pane. -->
+Откройте это [демо](https://ng-devtools.netlify.app/). Перейдите на вкладку **Источники** (Sources). `node_modules` и `webpack` — сторонние скрипты. Нажмите на иконку меню  &nbsp; {% Img src="image/admin/4sdCQbpBaG4MpoHB1J08.png", alt="More", width="4", height="20" %} &nbsp; и выберите  **Скрыть источники из списка игнорируемых** (Hide ignore-listed sources) чтобы спрятать такие файлы из дерева навигации.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:134]:
```suggestion
Откройте это [демо](https://ng-devtools.netlify.app/). Перейдите на вкладку **Источники** (Sources). `node_modules` и `webpack` — сторонние скрипты. Нажмите на иконку меню  &nbsp; {% Img src="image/admin/4sdCQbpBaG4MpoHB1J08.png", alt="More", width="4", height="20" %} &nbsp; и выберите  **Скрыть источники из списка игнорируемых** (Hide ignore-listed sources), чтобы спрятать такие файлы из дерева навигации.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 14]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Y4KSjl9zJQdnAhTvtnXm.png", alt="Hiding ignore-listed sources in the Sources panel", width="800", height="449" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Y4KSjl9zJQdnAhTvtnXm.png", alt="Скрытые источников из списка игнорируемых во вкладке Источники (Sources)", width="800", height="449" %}
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:136]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Y4KSjl9zJQdnAhTvtnXm.png", alt="Скрытие источников из списка игнорируемых во вкладке Источники (Sources)", width="800", height="449" %}
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 15]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
-Chromium bug: [1352488](https://crbug.com/1352488)
+Задача в трекере Chromium: [1352488](https://crbug.com/1352488)
 
 
 <!-- ## Hiding ignore-listed files in the Command Menu {: #ignore-search } -->
+## Скрытие игнорируемых файлов из Меню Команд (Command Menu) {: #ignore-search }
 
 <!-- With the **hide ignore-listed sources** setting, you can find your file quicker with the [Command Menu](/docs/devtools/command-menu/). Previously, searching files in the **Command Menu** returns third-party files that might not be relevant to you. -->
+Благодаря настройке  **Скрыть источники из списка игнорируемых** (Hide ignore-listed sources) вы можете быстрее искать свои файлы в [Меню Команд](/docs/devtools/command-menu/). Раньше поиск в ** Меню Команд** (Command Menu) показывал сторонние файлы, которые моглы быть не релевантны для вас.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:145]:
```suggestion
Благодаря настройке  **Скрыть источники из списка игнорируемых** (Hide ignore-listed sources) вы можете быстрее искать свои файлы в [Меню Команд](/docs/devtools/command-menu/). Раньше поиск в ** Меню Команд** (Command Menu) показывал сторонние файлы, которые могли быть нерелевантны для вас.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 16]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
-Chromium bug: [1336604](https://crbug.com/1336604)
+Задача в трекере Chromium: [1336604](https://crbug.com/1336604)
 
 
 <!-- ## New Interactions track in the Performance panel  {: #performance } -->
+## Новый трек Взаимодействия (Interactions) во вкладке Производительность (Performance) {: #performance }
 
 <!-- Use the new **Interactions** track in the **Performance** panel to visualize interactions and track down potential responsiveness issues.  -->
+Используйте новый трек **Взаимодействия** (Interactions) во вкладке **Производительность** (Performance) чтобы визуализировать взаимодействия и отследить потенциальные проблемы с отзывчивостью.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:162]:
```suggestion
Используйте новый трек **Взаимодействия** (Interactions) во вкладке **Производительность** (Performance), чтобы визуализировать взаимодействия и отследить потенциальные проблемы с отзывчивостью.
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 17]
```diff
@@ -5,14 +5,13 @@ authors:
   - jecelynyeen
 date: 2022-09-16
 updated: 2022-09-16
-description: "Better support for modern web debugging, LCP timings breakdown in the Performance Insights, and more."
+description: "Улучшенна поддержка современных способов отладки, разбивка по времени LCP во вкладке Performance Insights и другое."
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:8]:
```suggestion
description: "Улучшена поддержка современных способов отладки, разбивка по времени LCP во вкладке Performance Insights и другое."
```

File: site/ru/blog/new-in-devtools-106/index.md [hunk 18]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
-Chromium bug: [1334585](https://crbug.com1334585)
+Задача в трекере Chromium: [1334585](https://crbug.com1334585)
 
 
 <!-- ### Automatically ignore known third-party scripts {: #auto-ignore } -->
+### Автоматическое игнорирование известных сторонних скриптов {: #auto-ignore }
 
 <!-- Identify issues in your code quicker during debugging because DevTools now automatically adds known third-party scripts to the ignore list. -->
+Быстрее находите ошибки в своём коде при отладке, теперь DevTools добавляют известные сторонные скрипты в список игнорируемых.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-106/index.md:83]:
```suggestion
Быстрее находите ошибки в своём коде при отладке благодаря тому, что теперь DevTools добавляет известные сторонние скрипты в список игнорируемых. 
```

File: site/ru/_partials/devtools/whats-new.md (hunk 1 above)
Comment (solarrust) [inline site/ru/_partials/devtools/whats-new.md:25]:
```suggestion
* [Скрытие источников из списка игнорируемых во вкладке Источники (Sources)](/ru/blog/new-in-devtools-106/#ignore-nav)
```

**Comment by solarrust** [comment]
@webmaxru thank you! And happy to "see" you again =)

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Thanks both!

File: site/ru/blog/new-in-devtools-106/index.md [hunk 19]
```diff
@@ -52,124 +58,156 @@ async increment() {
...
 
-Chromium bug: [1352488](https://crbug.com/1352488)
+Задача в трекере Chromium: [1352488](https://crbug.com/1352488)
 
 
 <!-- ## Hiding ignore-listed files in the Command Menu {: #ignore-search } -->
+## Скрытие игнорируемых файлов из Меню Команд (Command Menu) {: #ignore-search }
 
 <!-- With the **hide ignore-listed sources** setting, you can find your file quicker with the [Command Menu](/docs/devtools/command-menu/). Previously, searching files in the **Command Menu** returns third-party files that might not be relevant to you. -->
+Благодаря настройке  **Скрыть источники из списка игнорируемых** (Hide ignore-listed sources) вы можете быстрее искать свои файлы в [Меню Команд](/docs/devtools/command-menu/). Раньше поиск в ** Меню Команд** (Command Menu) показывал сторонние файлы, которые могли быть нерелевантны для вас.
```
Comment (jecfish) [inline site/ru/blog/new-in-devtools-106/index.md:145]:
```suggestion
Благодаря настройке  **Скрыть источники из списка игнорируемых** (Hide ignore-listed sources) вы можете быстрее искать свои файлы в [Меню Команд](/docs/devtools/command-menu/). Раньше поиск в **Меню Команд** (Command Menu) показывал сторонние файлы, которые могли быть нерелевантны для вас.
```

**Pull request merged**